crawler/crawler.py

```diff
@@ -165,10 +173,10 @@ def main():
     parser.add_argument(
         '--features',
         dest='features',
-        type=str,
-        default=defaults.DEFAULT_FEATURES_TO_CRAWL,
+        type=csv_list,
+        default=get_config()['general']['features_to_crawl'],
```

Comment:
R1 + for clarity, maybe have a wrapper get_default() that reads from config_spec_and_defaults.conf, and let get_config() read from crawler.conf?